ros2bag/ros2bag/verb/record.py

```diff
@@ -174,8 +178,10 @@ def main(self, *, args):  # noqa: D102
         if (args.all and (args.topics or args.regex)) or (args.topics and args.regex):
             return print_error('Must specify only one option out of topics, --regex or --all')
         # one out of "all", "topics" and "regex" must be true
-        if not(args.all or (args.topics and len(args.topics) > 0) or (args.regex)):
-            return print_error('Invalid choice: Must specify topic(s), --regex or --all')
+        if not(args.all or (args.topics and len(args.topics) > 0) or (args.regex) or \
```

Comment:
```suggestion
        if not(args.all or (args.topics and len(args.topics) > 0) or (args.regex) or
```
This is what's causing the failing test.

Reply:
Ah yeah, I forgot to run linting beforehand. Thanks!